howso/react_aggregate.amlg

```diff
@@ -760,4 +849,84 @@
 			))
 		)
 	)
+
+	;Helper method for ReactAggregate that checks the output to see if the requested stat exists
+	#!ValidateCachedStats
+	(declare
+		(assoc
+			features (list)
+			stat_name (null)
+			selected_prediction_stats (null)
+		)
+
+		(declare (assoc exists (true)))
+
+		;prediction stats validation depends on the selected prediction stats
+		(if (= stat_name "prediction_stats")
+			(if
+				;if mae is the only prediction stat only, make sure every feature has a mae value
+				(and
+					(= (size selected_prediction_stats) 1)
+					(= (first selected_prediction_stats) "mae")
+				)
+				(seq
```

Comment:
This pattern of
`(seq (map (lambda ... (assign (assoc exists (false))) )  features) exists)`
is a bit complicated. 

I suggest instead just doing a map that retrieves it all (dropping the `(not)`) and wrapping the map in a `(contains_value (map ...) (null))`. That way we get a boolean value which will be (true) if the stat is missing for any feature. Which we can wrap in a (not) to be more like what youre aiming at.